.github/workflows/test.yml

```
@@ -0,0 +1,133 @@
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        pip install 'requests>=1.0' 'python-dateutil>=2.1' 'six>=1.2.0'
+        pip install pytest-cov python-coveralls
+        pip install -e .
+    - name: Lint with flake8
+      continue-on-error: true
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+
+  functional:
+    needs: lint
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        pip install 'requests>=1.0' 'python-dateutil>=2.1' 'six>=1.2.0'
+        pip install pytest-cov python-coveralls
+        pip install -e .
+    - name: Install mock on py2.7
+      if: ${{ matrix.python-version == '2.7' }}
+      run: |
+        pip install mock
+    - name: Test with pytest
+      run: |
+        pytest tests/functional --cov=pyuploadcare
+
+  django:
+    needs: functional
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
+        # this is a bit excessive, probably should run only before
+        #   releasing new version
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        django-version: [2.2, 3.0, 3.1, 3.2]
+        include:
+          - python-version: 2.7
+            django-version: 1.11
+          - python-version: 3.7
+            django-version: 1.11
+          - python-version: 3.5
+            django-version: 2.2
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        pip install 'requests>=1.0' 'python-dateutil>=2.1' 'six>=1.2.0'
+        pip install pytest-cov python-coveralls
```

`flake8` is not required. Others installed with `pip install -e .`

```suggestion
        pip install pytest pytest-cov python-coveralls
```